src/main/java/seedu/address/logic/commands/DeleteAssignmentCommand.java

```diff
@@ -26,6 +32,7 @@ public class DeleteAssignmentCommand extends Command {
             + "Parameters: "
             + PREFIX_NAME + "STUDENT_NAME "
             + PREFIX_ASSIGNMENT + "ASSIGNMENT "
+            + PREFIX_STUDENT_NUMBER + "STUDENT NUMBER (OPTIONAL)"
```

Comment:
![image](https://github.com/user-attachments/assets/d895f041-1f7f-4d37-94ed-0e8b7907167a)
Remember to add whitespaces in between the (OPTIONAL) and the next command